testkit/src/api.rs

```diff
@@ -41,6 +43,37 @@ pub enum ApiKind {
     Service(&'static str),
 }
 
+#[derive(Debug)]
+struct TestkitApiError(ApiError);
```

Comment:
Unless I'm misunderstanding something, this wrapper is superfluous; you could use a custom function (instead of an `Into` trait) in order to convert `Response` to `ApiError`.